### DISCUSSION TITLE
Arbitrary pixel values get stripped from production release

### DISCUSSION THREAD

**Discussion by bjornharvold** [tailwindlabs/tailwindcss#8131] [Help] [answered]
Hi TailwindCSS Community!

We just released our first TailwindCSS app to production and we noticed it didn't look like it should. 

App:
Angular 13.x / Nx 13.x

FYI
We have a prefix "kick-" in our tailwind-config.js.

When we use a padding like this: p-[10px], it would show up in our local environment CSS as kick-p-\[10px\]. 

When we deploy to production, that style does not exist in our stylesheet. It's been stripped out. As a result, the padding is not applied.

Not sure who the culprit is so starting our efforts in this forum.

See attached images of screenshots from local vs prod.

<img width="1583" alt="Screen Shot 2022-04-17 at 12 42 02 PM" src="https://user-images.githubusercontent.com/641791/163702305-1a79dc72-61d7-41c0-ad58-53522f2874c2.png">
<img width="1583" alt="Screen Shot 2022-04-17 at 12 41 18 PM" src="https://user-images.githubusercontent.com/641791/163702310-f9da2001-b005-4265-9048-6731a0951942.png">

**Comment by adamwathan** [comment, marked as answer]
Hey! Can you create a small project that reproduces the issue and share it as a GitHub repo so we can take a look?

**Comment by bjornharvold** [comment]
Hi @adamwathan 

I've had some more time to look into this. I am going to close this ticket and open a new one starting with the nx.dev team.

🍺
